src/Commands/sql/SqlCommands.php

```diff
@@ -142,8 +143,11 @@ public function cli(InputInterface $input, $options = ['extra' => self::REQ])
     {
         $sql = SqlBase::create($options);
         $process = $this->processManager()->shell($sql->connect(), null, $sql->getEnv());
-        $process->setTty($this->getConfig()->get('ssh.tty', $input->isInteractive()));
-        $process->setInput(STDIN);
+        if (!TerminalUtils::stdinIsTerminal(false)) {
```

Comment:
@weitzman Do you think it would be worthwhile to simplify this with a $process helper?
```
$process->setDefaultRedirectBehavior(STDIN, $input, $this->getConfig()->get('ssh.tty'));
```
Then the TerminalUtils equivalent moves into the site-process library.

I tend to think this is better. Thinking about it.

Reply:
Decided against the wrapper function for now. Maybe later.